### PR TITLE
Improve Convert2RHEL [priority]

### PR DESCRIPTION
**Pull request by Lennonka**
A couple of formal improvements.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [x] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [x] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/common/modules/con_converting-a-host-to-rhel.adoc [hunk 1]
```diff
@@ -31,14 +31,15 @@ The duration of the conversion process depends on the number of packages that ha
...
 Alternatively, you can use Ansible variables to tell the role to import the manifest from disk.
 The manifest must be imported to the organization to which you will register hosts for conversion.
 +
 You can update your allocations and download the updated manifest from the https://access.redhat.com[Red Hat Customer Portal].
 For more information, see https://access.redhat.com/documentation/en-us/red_hat_subscription_management/2022/html/using_red_hat_subscription_management/using_manifests_con[Using manifests] in _Red Hat Subscription Management_.
-* Ensure that you have enabled Red Hat repositories in {Project} for the minor {RHEL} version to which you convert your hosts.
+* Ensure that you have enabled and synchronized Red Hat repositories in {Project} for the minor {RHEL} version to which you convert your hosts.
+For more information, see {ContentManagementDocURL}Enabling_Red_Hat_Repositories_content-management[Enabling Red Hat Repositories] and {ContentManagementDocURL}Synchronizing_Repositories_content-management[Synchronizing Repositories] in _{ContentManagementDocTitle}_.
```
Comment (maximiliankolb) [inline guides/common/modules/con_converting-a-host-to-rhel.adoc:42]: Will two links in one line work? I remember there was an issue about this in the past.

**Review by maximiliankolb — COMMENTED** [review]
change looks good to me.

File: guides/common/modules/con_converting-a-host-to-rhel.adoc (hunk 1 above)
Comment (Lennonka) [reply guides/common/modules/con_converting-a-host-to-rhel.adoc:42]: The problem was with xrefs. Normal links should be fine. (I hope)

**Comment by Lennonka** [comment]
Cherry-picked:
- a8894f80c..7cd4ef898  3.8 -> 3.8
- 03d0990da..f072bb282  3.7 -> 3.7
- 2218c7aa9..66dbac307  3.6 -> 3.6
- 5d70f7a12..d523cdb08  3.5 -> 3.5

There was a conflict on 3.4 and I've decided not to cherry-pick further.

**Pull request merged**